Examples/VaporExample/README.md

```diff
@@ -1,13 +1,70 @@
-# VaporExample
```

Comment:
I think a lot of this text would be really good to include in the Vapor extension README too, since that's likely another visible place for it. Maybe we ought to have an abbreviated version of this README with small code snippets instead of a full example and then link to this one for the more detailed example?